### ISSUE TITLE
Subscriptions with watchedAttributes do not return complete entity

### ISSUE THREAD

**Issue by idomingu** [ScorpioBroker/ScorpioBroker#309]
Hi everyone,

I have noticed that subscriptions configured with `watchedAttributes` are sending notifications that do not contain the complete entity, just the watched attribute. As per ETSI CIM 009, if subscription does not configure `notification.attributes`, then all entity attributes shall be included.

**Environment**
- scorpiobroker/scorpio:gateway_aaio_2.1.13
- scorpiobroker/scorpio:eureka_aaio_2.1.13
- scorpiobroker/scorpio:scorpio-aaio_2.1.13

**Steps to reproduce**
1. Create example `Vehicle` entity:
```bash
curl --location --request POST 'http://localhost:9090/ngsi-ld/v1/entities/' \
--header 'Content-Type: application/json' \
--header 'Link: <https://fiware.github.io/data-models/full-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
--data-raw '{
	"id": "urn:ngsi-ld:Vehicle:A100",
	"type": "Vehicle",
	"brandName": {
		"type": "Property",
		"value": "Mercedes"
	},
	"speed": {
		"type": "Property",
		"value": 80
	}
}'
```
2. Create subscription for updates on `speed` attribute of `Vehicle` entity type:
```bash
curl --location --request POST 'http://localhost:9090/ngsi-ld/v1/subscriptions/' \
--header 'Content-Type: application/json' \
--header 'Link: <https://fiware.github.io/data-models/full-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
--data-raw '{
	"type": "Subscription",
	"entities": [{
		"type": "Vehicle"
	}],
    "watchedAttributes": ["speed"],
	"notification": {
		"endpoint": {
			"uri": "https://consumer/notify",
			"accept": "application/json"
		}
	}
}'
```
3. Update value of `speed` attribute of `urn:ngsi-ld:Vehicle:A100`:
```bash
curl --location --request PATCH 'http://localhost:9090/ngsi-ld/v1/entities/urn:ngsi-ld:Vehicle:A100/attrs' \
--header 'Content-Type: application/json' \
--header 'Link: <https://fiware.github.io/data-models/full-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
--data-raw '{
    "speed": {
        "type": "Property", 
        "value": 120
    }
}'
```
4. Notification received at `https://consumer/notify` contains the entity only with `speed` attribute.
```json
{
    "id": "ngsildbroker:notification:-7498080897937598466",
    "type": "Notification",
    "subscriptionId": "urn:ngsi-ld:Subscription:1495927664",
    "notifiedAt": "2022-06-02T07:49:15Z",
    "data": [
        {
            "id": "urn:ngsi-ld:Vehicle:A100",
            "type": "Vehicle",
            "speed": {
                "type": "Property",
                "value": 120
            }
        }
    ]
}
```

Thanks!

**Comment by ScorpioBroker** [comment]
Hi,
thanks, definitely a bug on our side. Will check

**Comment by amitrnec** [comment]
Hi,
Currently working on this issue.

Thanks.

**Comment by amitrnec** [comment]
Hi,
We have fixed this issue #314 

Thanks.

**Comment by ScorpioBroker** [comment]
Hi, dockerhub version 2.1.15 has the bugfix for this.
